src/ansible_navigator/runner/ansible_inventory.py

```diff
@@ -27,26 +27,18 @@ def fetch_inventory(
     ) -> Tuple[str, str]:
         """Run ansible-inventory command and get the inventory related details
 
-        Args:
-            action (str): Valid values are one of ``graph``, ``host``, ``list`` \
-                          ``graph`` create inventory graph, ``host`` returns specific \
```

Comment:
I usually prefer this format of indentation (not talking about the syntax) because it makes the visual distinction between the param and its description.

Also, in places with EOL escaping like this, it's usually necessary to turn the docstring into a raw-string (`r""" stuff """` — pay attention at the r-preffix).

Reply:
I think we talked about this and my pref for a single tab which keeps the indenting common and allows for fewer lines given the extra spaces.  I think both are fine, so it will be a long time before I might do any global changes across everything :)